### ISSUE TITLE
Notifications about comments, likes and new posts / chats

### ISSUE THREAD

**Issue by pwFoo** [irislib/iris-messenger#54]
A page with all reactions like comments, likes, ... about my own account?

**Comment by mmalmi** [comment]
Definitely a needed core feature 👍

**Comment by trosel** [comment]
yeah each user should have an inbox where all these events are sent to

**Comment by trosel** [comment]
i just realized that this already exists. i think it’s just missing a “badge” that shows you when you have new notifications. 

https://iris.to/notifications

**Comment by petrikaj** [comment]
Iris is a nostr client now.
